autoRSA.py

```diff
@@ -2,7 +2,6 @@
 # Script to automate RSA stock purchases
 
 # Import libraries
-import asyncio
```

Comment:
Why is asyncio removed? It's used on line 356

Reply:
My IDE said it was an unused dependency and it runs fine without importing it. 
![image](https://github.com/user-attachments/assets/05fb0c42-3cac-4d08-97e2-749e011e8e1a)
![image](https://github.com/user-attachments/assets/290fb051-e338-43a5-b0e5-94568666a66d)

Keep in mind, for these screenshots I did not import asyncio when running the asyncio line

Reply:
Huh, my code runs fine but my editor (VS Code) stops offering code completion for asyncio without the import.